Nimble/objc/DSL.h

```diff
@@ -107,6 +107,13 @@ NIMBLE_EXPORT id<NMBMatcher> NMB_allPass(id matcher);
 NIMBLE_SHORT(id<NMBMatcher> allPass(id matcher),
              NMB_allPass(matcher));
 
+NIMBLE_EXPORT id<NMBMatcher> NMB_satisfyAnyOfWithNilTermination(id matcher, ...)
+    NS_REQUIRES_NIL_TERMINATION;
+#define NMB_satisfyAnyOf(...) NMB_satisfyAnyOfWithNilTermination(__VA_ARGS__, nil)
```

Comment:
Nitpick: It might be better to use an array literal in the macro instead of nil termination. This gives us a compiler / runtime error when nil is passed along. IIRC, nil termination short-circuits if an earlier `nil` is given in the arg list.